cli/src/runtime.rs

```diff
@@ -118,7 +118,21 @@ pub async fn run(ad4m_client: Ad4mClient, command: RuntimeFunctions) -> Result<(
         }
         RuntimeFunctions::HcAgentInfos => {
             let infos = ad4m_client.runtime.hc_agent_infos().await?;
-            println!("{}", infos);
+            println!("RAW\n{}\n\n", infos);
+            let encoded_agent_infos: Vec<String> = serde_json::from_str(&infos)?;
```

Comment:
This decoding is just me testing the serialization. I think we can just print out the string here (or maybe deserialize it just to print a pretty list with agent addresses), but we need this code in the resolver for `HcAddAgentInfos`.